src/test/java/com/iota/iri/service/tipselection/impl/CumulativeWeightCalculatorTest.java

```diff
@@ -283,7 +283,6 @@ public void testTangleWithCircle2() throws Exception {
     }
 
     @Test
-    @Ignore
```

Comment:
Are we using the prefix again?
How come this is passing?

When I asked before I just didn't check what this unit test does.
Then you reminded and convinced me that this should be ignored and probably deleted later once we finalize tipsel.